### ISSUE TITLE
Python3 Support

### ISSUE THREAD

**Issue by chasehobbs** [genesisfactor/DreamScreenCommander#3]
Many systems are moving to Python3 with the deprecation of 2. Had to make a few changes to get this to run with Python3. Mostly around the print statements and the IP/ip variables.

**Comment by genesisfactor** [comment]
you're absolutely right, @chasehobbs .  Could you share your changes in a PR?  Among things, I hadn't set up in my Dreamscreen since I moved and went to 4K, so it is difficult for me to test right.

**Comment by D3ltorohd** [comment]
Hello, i hat the same problem. I have found some infos in an forum. So what to do is. All lines with print "...." put into ( )

print("Text here") 

After this, all works fine, without errors on pyhthon 3. Tested on my system. So you can change your code.
